### DISCUSSION TITLE
slowness opening albums

### DISCUSSION THREAD

**Discussion by Baltimorepc** [LycheeOrg/Lychee#2769] [Q&A]
ive been having issues when opening albums. Sits at a black screen and then never loads the album. OR takes a very long time. Is anyone else having this issue and how can i fix it.

**Comment by ildyria** [comment]
Which version are you on?
Can you give us your diagnostic?
Did you go to the maintenance page and click optimize DB ?

**Comment by Baltimorepc** [comment]
I just installed the latest thinking it help but didn’t and yes I did run the optimize database 


Sent from Yahoo Mail for iPhone


On Sunday, December 1, 2024, 1:37 PM, Benoît Viguier ***@***.***> wrote:




Which version are you on?
Can you give us your diagnostic?
Did you go to the maintenance page and click optimize DB ?

**Comment by ildyria** [comment]
Can you set `VUEJS_ENABLED=false` in your `.env` just to check?
Also how many albums do you have?

**Comment by Baltimorepc** [comment]
i only have 4 albums.
i can try to add that variable.

**Reply by ildyria** [reply]
It is very very surprising it is that slow.
Can you check what is taking so long in your Network tab?

**Comment by Baltimorepc** [comment]
im actually home and accessing it is still very slow when opening albums. sometimes it will eventually load. other times it just sits at a black screen. and chrome doesnt even show that the page is loading still.

**Reply by ildyria** [reply]
Please pick one of the response and reply directly in it, not at the bottom of this discussion page. Otherwise it makes a mess in the potential answers.

That is very very surprising. I am loading my page in 1,4 s. which I will admit is slow but still. 
![image](https://github.com/user-attachments/assets/be9be6db-bad7-40d3-a47f-23efcf0f9edc)
